packages/db/src/resources/bytecodes.ts

```diff
@@ -13,7 +14,7 @@ export const bytecodes: Definition<"bytecodes"> = {
       id: ID!
       bytes: Bytes!
       linkReferences: [LinkReference]
-      instructions: [Instruction!]
+      instructions(count: Int): [Instruction!]
```

Comment:
I'm not clear on why this is a function.  (Am I reading this right?  It's a function?)  I assume that has something to do with `parseCode` currently taking a count, but, like, this still shouldn't be a function, right?  It should be, like, the right thing.

Perhaps it would make sense to hold off on this PR until we change `parseCode`?  I could do that pretty quickly assuming we agree on what course of action to take.

Reply:
This is just GraphQL syntax - since it's an optional argument (as opposed to `count: Int!`), you would query as:
```graphql
{
  bytecodes {
    instructions {
      programCounter
    }
  }
}
```

or if wanted to specify a count, then you'd use parens: 

```graphql
{
  bytecodes {
    instructions(count: 5) {
      programCounter
    }
  }
}
```

GraphQL just doesn't make the distinction like you'd see in JS

Reply:
I see!  OK, that seems fine then, but in that case I think we should make the change in #3528 and take approach (3) so that we don't accidentally strip metadata on bytecode that lacks it.  Do you want me to do that?

Reply:
sure, sounds good!

Reply:
Oh, wait, sorry, further question about this.  I'm pretty sure that this is correct, but, just to be certain: This is bytecode as output by the compiler, right?  So e.g. for constructor bytecode it will never have arguments attached, right?

Reply:
correct

Reply:
Ah crap actually, I just realized -- stripping the metadata the way I was hoping doesn't actually work because there can be more metadata than just that.  You know what, this is fine, let's ignore it.  We can worry about this at a later time.